example_cpp_smart_card_client_app/src/background.js

```diff
@@ -109,8 +109,7 @@ const MAIN_WINDOW_OPTIONS = {
   'hidden': true,
   'id': MAIN_WINDOW_ID,
   'innerBounds': {
```

Comment:
Should it become just `'width': 500`, without nesting inside innerBounds? Your new typedef doesn't have innerBounds anymore.

Reply:
so innerBounds doesn't need to be defined?